requirements.txt

```diff
@@ -1,5 +1,5 @@
 # 3.9.1 does not work with Python 3.11 and 3.12
-aiohttp==3.9.0
+aiohttp==3.9.5
```

Comment:
3.9.0 contains some known security vulnerabilities which are fixed in newer versions, but the comment above makes it sound like we should actually not update.
Should we maybe first bump the used Python version in the GitHub Action workflow and then see whether this PR still passes with newer Python versions before we merge this?
cc @pm-osc

Reply:
Thanks @FlorianHockmann for checking this. I have tested manually and aiohttp 3.9.5 works fine also with Python 3.11.9 and 3.12.3. I have checked and aiohttp 3.9.1 also works with the mentioned newer Python versions so back then I might have overlooked something when I added the mentioned comment.

Reply:
Great, thanks for investigating. Then I'll remove the comment here and merge the update.